pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This PR is about the removal of an obsolete dependency and not the introduction of another build system.